### PR TITLE
Shopfloor misc imp/fix

### PR DESCRIPTION
**Pull request by simahawk**
Major changes:

* fix/imp all info screens
* split user/app services
* refactor headers validation to services
* add picking edit form
* fix mobile manifest (prevent add to home screen)

See atomic commits for more details.

Ref: 1700, 1719, 1683

### CODE REVIEW

File: shopfloor_mobile/static/wms/src/services/odoo.js [hunk 1]
```diff
@@ -67,16 +67,19 @@ export class OdooMixin {
...
         // topics/tutorials/config_pitfalls/#missing-disappearing-http-headers
-        return {
-            "Content-Type": "application/json",
+        let headers = {
             "SERVICE-CTX-MENU-ID": this.process_menu_id,
             "SERVICE-CTX-PROFILE-ID": this.profile_id,
             "API-KEY": this.apikey,
         };
+        if (method == "POST") {
+            headers["Content-Type"] = "application/json";
```
Comment (guewen) [inline shopfloor_mobile/static/wms/src/services/odoo.js:80]: Why this condition? Are we returning something else than json on `GET`?

File: shopfloor_mobile/static/wms/src/scenario/form.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+            }
+            const self = this;
+            let form_values = form_data.values;
+            form_values["_id"] = this.record.id;
+            this.odoo.call("update", form_values).then(function(result) {
+                self._load_form_data(result);
+                setTimeout(function() {
+                    self.$root.trigger("go_back");
+                    self.$router.back();
+                }, 1500);
```
Comment (guewen) [inline shopfloor_mobile/static/wms/src/scenario/form.js:68]: is there a rationale for this duration?

File: shopfloor_mobile/static/wms/src/services/odoo.js (hunk 1 above)
Comment (simahawk) [reply shopfloor_mobile/static/wms/src/services/odoo.js:80]: this is what we send and on GET we don't send any JSON.

File: shopfloor_mobile/static/wms/src/scenario/form.js (hunk 2 above)
Comment (simahawk) [reply shopfloor_mobile/static/wms/src/scenario/form.js:68]: bah, it's just a trick to get the success message displayed and then redirect to where you were before.
It should be improved somehow by passing the message to parent component.
I'll add a TODO.

Comment (simahawk) [reply shopfloor_mobile/static/wms/src/scenario/form.js:68]: done

**Review by sebalix — APPROVED** [review]

**Pull request merged**